lib/browser/api/utility-process.ts

```diff
@@ -1,56 +1,20 @@
 import { EventEmitter } from 'events';
-import { Readable } from 'stream';
-import * as path from 'path';
-import { app } from 'electron/main';
+import { PassThrough } from 'stream';
+import { Socket } from 'net';
 import { MessagePortMain } from '@electron/internal/browser/message-port-main';
-const { createProcessWrapper } = process._linkedBinding('electron_browser_utility_process');
+const { _fork } = process._linkedBinding('electron_browser_utility_process');
 
-class IOReadable extends Readable {
-  #shouldPush: boolean = false;
-  #data: (Buffer | null)[] = [];
-  #resume: (() => void) | null = null;
-
-  storeInternalData (chunk: Buffer | null, resume: (() => void) | null) {
-    this.#resume = resume;
-    this.#data.push(chunk);
-    this.#pushInternalData();
-  }
-
-  #pushInternalData () {
-    while (this.#shouldPush && this.#data.length > 0) {
-      const chunk = this.#data.shift();
-      this.#shouldPush = this.push(chunk);
-    }
-    if (this.#shouldPush && this.#resume) {
-      const resume = this.#resume;
-      this.#resume = null;
-      resume();
-    }
-  }
-
-  _read () {
-    this.#shouldPush = true;
-    this.#pushInternalData();
-  }
-}
-
-export default class UtilityProcess extends EventEmitter {
+class ForkUtilityProcess extends EventEmitter {
   #handle: ElectronInternal.UtilityProcessWrapper | null;
-  #stdout: IOReadable | null | undefined = new IOReadable();
-  #stderr: IOReadable | null | undefined = new IOReadable();
-  constructor (modulePath: string, args?: string[], options?: Electron.UtilityProcessConstructorOptions) {
+  #stdout: any = null;
+  #stderr: any = null;
```

Comment:
Sad that these became `any`, can we refine the type again please?

Reply:
Refs https://github.com/electron/electron/pull/34980/commits/a82d3223ede85df14b97780ef2bc1cce21e3aa39